torch/csrc/autograd/python_variable.cpp

```diff
@@ -2012,7 +2009,7 @@ void THPVariable_subclass_dealloc(PyObject* self) {
 // It's ALWAYS safe (albeit slower) to call this with MAYBE_UNINITIALIZED.
 static PyObject* THPVariable_NewWithVar(
     PyTypeObject* type,
-    Variable _var,
+    const at::TensorBase& _var,
```

Comment:
Why is this changed to a TensorBase from a Variable?

Reply:
A chain of interface changes made me found that it actually only needs  const at::TensorBase&